client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2350,6 +2350,34 @@ Undocumented.prototype.transferStatus = function( siteId, transferId ) {
 };
 
 /**
+ * Update the poster for a video.
+ *
+ * @param {string} videoId ID of the video
+ * @param {object} data The POST data
+ * @param {Function} fn Function to invoke when request is complete
+ * @returns {Promise} A promise that resolves when the request is complete
+ */
+Undocumented.prototype.updateVideoPoster = function( videoId, data, fn ) {
```

Comment:
I'll very much encourage using the new data layer for any and all new wpcom endpoints and would be happy to walk you through it. In the meantime, check out the [API layer](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/README.md) and [HTTP layer](https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom-http/README.md) readmes.